### PR TITLE
Use transfomers tokenizer and streamer for python api

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

feature
API changed

- [x] finish TODO of https://github.com/intel/intel-extension-for-transformers/pull/252
- [x] refine setup.py

## Expected Behavior & Potential Risk

```python
from transformers import AutoTokenizer, TextStreamer
from intel_extension_for_transformers.transformers import AutoModel, WeightOnlyQuantConfig

model_name = "THUDM/chatglm2-6b"  # or local path to model
woq_config = WeightOnlyQuantConfig(compute_dtype="int8", weight_dtype="int4")
prompt = "She opened the door and see"

tokenizer = AutoTokenizer.from_pretrained(model_name, trust_remote_code=True)
input_ids = tokenizer(prompt, return_tensors="pt").input_ids
streamer = TextStreamer(tokenizer)

model = AutoModel.from_pretrained(model_name, quantization_config=woq_config, use_llm_runtime=True, trust_remote_code=True)
gen_tokens = model.generate(input_ids, streamer=streamer, max_new_tokens=300)
```

### CODE REVIEW

File: intel_extension_for_transformers/llm/runtime/graph/__init__.py [hunk 1]
```diff
@@ -48,30 +48,31 @@ def __import_package(self, model_name):
             import intel_extension_for_transformers.llm.runtime.graph.opt_cpp as cpp_model
         elif model_name == "bloom":
             import intel_extension_for_transformers.llm.runtime.graph.bloom_cpp as cpp_model
-        elif model_name == "chatglm2":
+        elif model_name == "chatglm":
```
Comment (hshen14) [inline intel_extension_for_transformers/llm/runtime/graph/__init__.py:51]: both chatglm and chatgml2?

Comment (zhenwei-intel) [reply intel_extension_for_transformers/llm/runtime/graph/__init__.py:51]: which is fixed in https://github.com/intel/intel-extension-for-transformers/pull/382, I will merge PR382

**Review by zhentaoyu — APPROVED** [review]

**Comment by kevinintel** [comment]
please update readme

**Comment by zhenwei-intel** [comment]
> please update readme

waiting for [PR#340](https://github.com/intel/intel-extension-for-transformers/pull/340)

**Comment by hshen14** [comment]
> > please update readme
> 
> waiting for [PR#340](https://github.com/intel/intel-extension-for-transformers/pull/340)

340 is done

**Review by hshen14 — APPROVED** [review]

**Review by kevinintel — APPROVED** [review]

**Review by a32543254 — APPROVED** [review]
LGTM

**Comment by VincyZhang** [comment]
windows build: https://inteltf-jenk.sh.intel.com/job/nlp-release-build-win/253/
generation extension tests: https://inteltf-jenk.sh.intel.com/job/ITREX-1.2-release-test/42/

**Pull request merged**